src/applications/simple-forms/40-0247/pages/requestType.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+
+const labelText =
+  'Is this your first time requesting a Presidential Memorial Certificate?';
+
+/* @type {PageSchema} */
+export default {
+  uiSchema: {
+    isFirstRequest: {
+      'ui:title': labelText, // yesNo widget doesn't support JSX
+      'ui:widget': 'yesNo',
```

Comment:
same

Reply:
Refactored to use v3 yesNoUI & yesNoSchema, but couldn't customize error-message.